app/pb_kits/playbook/pb_distribution_bar/_distribution_bar.scss

```diff
@@ -0,0 +1,29 @@
+@import "../tokens/colors";
+
+[class^=pb_distribution_bar] {
+  $bar-colors: map-values($data_colors);
+  display: flex;
+  .pb_distribution_value {
+    height: 100%;
+    display: inline-block;
+    @for $i from 1 through length($bar-colors) {
+      &:nth-child(#{length($bar-colors)}n+#{$i}) {
+        background-color: nth($bar-colors, $i);
+      }
+    }
+    &:first-child {
+      border-top-left-radius: 18px;
+      border-bottom-left-radius: 18px;
+    }
+    &:last-child {
+      border-top-right-radius: 18px;
+      border-bottom-right-radius: 18px;
+    }
+  }
+  &[class*=_sm] {
```

Comment:
Does this class need to be this generic? It will match anything that that has a `_sm` in it. That means it would match `.pb_distribution_bar_smooth`. Seems like that is overly generic.